app/controllers/api/samples_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::SamplesController < ApplicationController
```

Comment:
it's good practice to separate APIs into versions (v1/ v2, etc.) to handle updates, so a sample controller should ideally be in /controllers/api/v1/samples_controller.rb